zerver/views/webhooks/gosquared.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from typing import Dict, Any
+
+BODY_TEMPLATE = '{website_name} has {user_num} visitors online.'
+
+@api_key_only_webhook_view('GoSquared')
+@has_request_variables
+def api_gosquared_webhook(request, user_profile, client,
+                          payload=REQ(argument_type='body'),
+                          stream=REQ(default='gosquared'),
+                          topic=REQ(default='GoSquared')):
```

Comment:
I'm wondering - maybe if the topic is not set by a user we should do it a little bit more precise? 
I mean maybe topic should be:
`GoSquared - request.bin` or something.